Services/alarmMetricService.js

```diff
@@ -0,0 +1,71 @@
+let model = require("../Models/metrics/alarm").alarm;
+
+let metricDB = require("knex")({

```

Comment:
Extract into a singleton, look at database.js and server.js. It connects on startup and all other services that use the postgres database will get the instance from the singleton